content/docs/guides/components/mpi.md

```diff
@@ -93,3 +88,57 @@ spec:
 status:
   launcherStatus: Succeeded
 ```
+
+
+Training should run for 100 steps and takes a few minutes on a GPU cluster. Logs can be inspected to see its training progress. Once everything starts, the logs can be accessed from the `launcher` pod:
```

Comment:
For improved readability, change the passive voice to active voice. So, change this:

```
Logs can be inspected to see its training progress. Once everything starts, the logs can be accessed from the `launcher` pod:
```

To this:

```
You can inspect the logs to see the training progress. When the job starts, access the logs from the `launcher` pod:
```